end-user-application-machines/new-agent-vm/rhel-standard-agent.json

```diff
@@ -251,6 +252,27 @@
                 "commandToExecute": "[concat('bash ', variables('scriptFileName'), ' \"', parameters('registrationCode'), '\" \"', parameters('agentType'), '\" \"', variables('vmSettings').vmNamePrefix, '\" \"', parameters('domainToJoin'), '\" \"', parameters('domainUsername'), '\" \"', parameters('domainPassword'), '\" \"', parameters('domainGroupToJoin'), '\"')]"
                 }
               }
-            }
+            },
+            {
```

Comment:
I didn't know you could run two script extensions - I guess that's new? Cool.

Reply:
Turns out you can't, I had to switch this to CustomScriptForLinux so it wouldn't clash with the other one